parsing/location.ml

```diff
@@ -897,6 +895,15 @@ let alert ?(def = none) ?(use = none) ~kind loc message =
 let deprecated ?def ?use loc message =
   alert ?def ?use ~kind:"deprecated" loc message
 
+let auto_include_alert lib =
+  let message = Printf.sprintf "\
+    OCaml's lib directory layout changed in 5.0. The %s subdirectory has been \
+    automatically added to the search path, but you should add -I +%s to the \
+    command-line to silence this alert." lib lib in
+  let alert =
+    {Warnings.kind="ocaml_auto_include"; use=none; def=none; message} in
```

Comment:
I am wondering if we want to name the alert `ocaml_deprecated_auto_include` to indicate that the alert may be replaced by an error in some future.

Reply:
Done - I've only changed the string, I assume the name of the function itself is OK without `_deprecated_` in it?